go.mod

```diff
@@ -157,7 +157,7 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kube-aggregator v0.0.0
 	k8s.io/kube-controller-manager v0.0.0
-	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
+	k8s.io/kube-openapi v0.0.0-20200409184531-690c19a814e8
```

Comment:
Let me know if you'd want me to tag that repo.

Reply:
I'm not sure what we should do... adding a v0.16.1 tag would make that the auto-selected version instead of master... which isn't great

Reply:
I think keeping the pseudo tag is the best we can do